draft-sharabayko-mops-srt.md

```diff
@@ -1226,7 +1226,11 @@ ACKACK control packets do not contain Control Information Field (CIF).
 
 A Message Drop Request control packet is sent by the sender to the receiver
 when it requests the retransmission of an unacknowledged packet (all or part
-of a message) which is not present in the sender's buffer (e.g. due to an internal error).
+of a message) which is not present in the sender's buffer. This may happen
+in the file/message mode when the message TTL parameter in `srt_sendmsg`
+was a defined timeout and the message was given up retransmission after
+this time, so the packet was removed from the sender buffer.
```

Comment:
- We don't have "file/message" mode defined in RFC.
- `srt_sendmsg` is library specific API function, not defined in the RFC. 

Consider generalizing this. Maybe, @stevomatthews , you could help here?